metals-sublime.sublime-settings

```diff
@@ -8,6 +8,7 @@
   "server_properties": [
     "-Dmetals.client=sublime",
     "-Dmetals.status-bar=on",
+    "-Dmetals.execute-client-command=on",
```

Comment:
So we moved a chunk of the server properties to `clientExperimentalCapabilities`. Do you know if it's possible to set them that way instead? In not this will still work, so it's not a huge deal.

Reply:
This would probably require a change in `LanguageHandler` API which this plugin is using. Possible but most likely in a separate PR.

Reply:
Sounds good. I'd say then just leave it as is, but since both the vs code an vim client have moved into that direction, it might be a good idea to do the same in the future.